File: Guia-2/src/Main.cpp
#include <iostream>
using namespace std;

class Aluno {
    double nota1, nota2, projeto;
  
    public:
        void setNotas(double,double,double,double, double);
        double getNotaFinal(){
            return (nota1 * 0.2) + (nota2 * 0.2) + (projeto * 0.6);
        } 

};

void Aluno::setNotas (double n11, double n12, double n21, double n22, double np) {
  nota1 = (n11 + n12) / 2;
  nota2 = (n21 + n22) / 2;
  projeto = np;
}


int main(int argc, char const *argv[]){

    Aluno aluno;

    aluno.setNotas(10,10,10,10,0);

    cout << "nota final: " << aluno.getNotaFinal() << "\n";  
    
    return 0;
}

File: Guia-2/src/Main.java
class Aluno {
    private Integer nota1;
    private Integer nota2;
    private Integer projeto;

    public Aluno(int guia11, int guia12,int guia21, int guia22, int projeto) {
        this.nota1 = (guia11 + guia12) / 2;
        this.nota2 = (guia21 + guia21) / 2;
        this.projeto = projeto;
    }
    public Double getNotaFinal(){
        Double notaFinal = this.nota1 * 0.2 + this.nota2 * 0.2 + this.projeto * 0.6;
        return notaFinal;
    }
}

public class Main {
    public static void main(String[] args) {
        Aluno aluno = new Aluno(10,10,10,10,0);

        System.out.println("Nota Final = " + aluno.getNotaFinal());
    }
}
